app/src/Containers/SearchContainer.tsx

```diff
@@ -307,7 +307,7 @@ const SearchContainer = () => {
         const fuse = new Fuse(projects, {
           keys: searchKeys,
           minMatchCharLength: 2,
-          threshold: 0.4,
+          threshold: 0.3,
```

Comment:
Just thinking about the conversation at the retro, could you add a comment here to say what `threshold` means and why it's set to 0.3?

Reply:
Good call Joe!

Reply:
The threshold was reduced because of the unwanted results, like the one you had for 'React'. I wonder if we should be using fuzzy search here at all? Agree this would be resolved if the skills field was standardised. Best to discuss that with Joanna.

Reply:
Yeah that's a good point @PammyLo -- looking at it again, maybe for the tech stack quick search it shouldn't use the fuzzy search and just look for the exact text instead, as these (Python, JavaScript, etc) are such specific terms.

Reply:
In fact, I can't remember a good reason why the quick search buttons need to use fuzzy search, so could generally simplify the quick search functionality and look for exact rather than fuzzy matches?  

(The benefit of fuzzy search is when people might misspell / write a word a little differently so it's useful for the free text search, but with the quick search buttons it's different - we set the search terms.  As long as text in the projects themselves is spelt the way we expect, an exact search could work here?)

Reply:
The fuzzy search is needed for the roles quick search and best avoided for the other quick searches. 
Thanks @joefhall for the comments for improvement and a huge thanks for our joint session to analyse what was needed here!

Reply:
Now that we're no longer adjusting how fuzzy search works to accommodate the skills quick search, I think it might be best if we leave this as it was at `0.4` as that setting seemed to work ok for other fields

Reply:
Are we ok to leave this as `0.4` instead of changing it to `0.3`?  I think we were only going to change this here because we were going to use fuzzy search for the skills / tech stack (which we're not doing now)

Reply:
sure, good idea, thanks Joe.